ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkTable/NetworkTable.tsx

```diff
@@ -170,6 +192,40 @@ const generateRows = (
             />
           ),
         },
+        {
+          content:
+            showSubnetLinks || showSubnetDisplay ? (
```

Comment:
Seems like there are a lot of ternaries in this cell, making it pretty difficult to know what will end up getting rendered. Maybe this could be its own component?

Reply:
Yeah I think you're right, I toyed with the idea but decided against it, but I think it'll be much easier to understand as a component.